environment.yml

```diff
@@ -39,8 +39,8 @@ dependencies:
   - sphinx-automodapi
   - git+https://github.com/iminuit/probfit.git
   - https://github.com/cta-sst-1m/protozfitsreader/archive/v1.4.0.tar.gz
-  - git+https://github.com/cta-observatory/ctapipe.git#egg=ctapipe
-  - git+https://github.com/cta-sst-1m/CTS.git#egg=CTS
-  - git+https://github.com/dstndstn/astrometry.net.git#egg=astrometry
+  - git+https://github.com/cta-observatory/ctapipe.git
+  - git+https://github.com/cta-sst-1m/CTS.git
+  - git+https://github.com/dstndstn/astrometry.net.git
   - https://github.com/calispac/histogram/archive/0.3.2.tar.gz
-  - git+https://github.com/uvemas/ViTables.git#egg=vitables
+  - git+https://github.com/uvemas/ViTables.git
```

Comment:
What was that `#egg` argument doing ?

Reply:
fom pip --help:
  --egg                       Install packages as eggs, not 'flat', like pip normally does. This option is not about installing *from* eggs. (WARNING: Because this option overrides pip's normal install logic, requirements files may not
                              behave as expected.)

Reply:
So it is good that we changed this ?

Reply:
I am not sure, my guess is that it should not have much impact.